dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -680,7 +682,13 @@ HRESULT CFSDropTarget::_DoDrop(IDataObject *pDataObject,
             op.hwnd = m_hwndSite;
             op.wFunc = bCopy ? FO_COPY : FO_MOVE;
             op.fFlags = FOF_ALLOWUNDO | FOF_NOCONFIRMMKDIR;
-            hr = SHFileOperationW(&op);
+            int res = SHFileOperationW(&op);
+            if (res)
```

Comment:
Replace `return hr;` with this block and `return S_OK;` too.

While here, `hr` at line 490/492 should be used or removed...

Reply:
OK to use it.